subsys/bluetooth/controller/ll_sw/ctrl.c

```diff
@@ -4895,10 +4915,12 @@ static inline void isr_radio_state_close(void)
 
 	event_inactive(0, 0, 0, NULL);
 
+#if defined(CONFIG_SOC_COMPATIBLE_NRF)
 	err = clock_control_off(_radio.hf_clock, NULL);
```

Comment:
have ```int err = ``` and remove the cond, compile above in the beginning of the function. Similarly at all other places.

Reply:
OK, that's doable